Library/Homebrew/formula_installer.rb

```diff
@@ -320,7 +321,7 @@ def install
     build_bottle_preinstall if build_bottle?
 
     unless @poured_bottle
-      build
+      build unless @fetched
```

Comment:
You only want to build if the build hasn't already been completed:

```suggestion
      build unless @built
```